rust/src/openvasd/scans/state_change.rs

```diff
@@ -64,14 +64,27 @@ pub(crate) fn row_to_models_status(scan_row: SqliteRow) -> models::Status {
     let dead = scan_row.get("host_dead");
     let alive = scan_row.get("host_alive");
     let finished = excluded + dead + alive;
+    let host_scanning: String = scan_row.get("host_scanning");
+
+    let host_progress: HashMap<String, i32> = host_scanning
```

Comment:
I know that it is a draft, but I would highly recommend to create a table for host_scanning so that we don't have to do this comma separation thingy everywhere and you don't need to alter a table.

```
CREATE TABLE scan_currently_running(
    id INTEGER,
    host TEXT NOT NULL,
    PRIMARY KEY (id, host),
    FOREIGN KEY (id) REFERENCES client_scan_map(id) ON DELETE CASCADE
);
```